docs/MapColonies/vector/Services/valhalla_service.md

```diff
@@ -0,0 +1,60 @@
+* `path` - the sequence of edges forming a route
+* `penalty` - fixed costs in seconds that are only added to the path cost. Penalties can influence the route path determination but do not add to the estimated time along the path.
+* `route` - sequence of edges and maneuvers forming the best travel path between locations given the available road network, costs, influence factors, and other inputs.
+* `short path` - a route that attempts to minimize distance traveled over the constituent edges, but may not be the shortest distance.
+* `through location` - an optional location to influence the route to travel through that location.
+* `tiled routing` - method of building a path on graph data that has been split into square cells.
+* `time` - the number of seconds estimated to complete a maneuver or trip, including any additional costs.
+* `trip` - results of an entire route, including locations, legs, and maneuvers.
+* height - with respect to elevation, the height above or below sea level at a specific location (lat,lng).
+* `height with range` - computing the range (cumulative distance) and height for a series of lat,lng pairs of a line or shape. This is very useful for charting/graphing.
+
+## Features
+1. Flexible routing methods with dynamic costing - allowing multiple route types with the same data.
+2. Expanded guidance / route narrative information providing clear and concise instructions along the route as well as output tailored for navigation along the route.
+3. Tiled routing data structure allowing use within small memory profiles including mobile devices.
+4. Includes software to import and create routing tiles from OpenStreetMap pbf data: from small extracts to worldwide coverage.
+5. Includes turn restrictions, highway exit information, administrative information, and timezones.
+
+## How to use
+
+Valhalla is an open-source toolkit for multimodal transportation, powered by open data. The APIs use the standard REST model of interaction with JSON serving as both the request and response formats. Please refer to a specific service's API documentation to learn about the various request/response properties. In addition to JSON request/response formats Valhalla also supports protocol buffers as request/response format.
+
+Its API has the following endpoints for querying the data:
+* __[route](/docs/MapColonies/openstreetmap/valhalla/turn-by-turn/osm-valhalla-route)__ - the service guides you between points by car, bike, foot, and multimodal combinations involving walking and riding public transit. Your apps can use the results from the route service to plan multimodal journeys with narratives to guide users by text and by voice. Valhalla draws data from OpenStreetMap for the main graph and from user-supplied GTFS feeds for multimodal routing. 
+    
+    #### "I am trying to run more than one errand in the day or start my own delivery service!" 🤷
```

Comment:
why is there a title in the middle of a bulleted list?
![image](https://github.com/MapColonies/developer-portal/assets/12687466/fb6451d5-6c48-47ec-9f9d-6e317113135b)